requirements.in

```
@@ -1,5 +1,5 @@
 -c api/runtime/requirements.txt
-aws_cdk.aws_dynamodb
-aws_cdk.pipelines
-aws_cdk.core
-cdk-chalice
+aws-cdk-lib>=2.0.0
```

This library doesn't have a version lower than 2.x, so this is a no-op and can confuse people to think why there is a version specifier here:
```suggestion
aws-cdk-lib
```

---

Removed no-op